MassEditConnector/Reader/ReferenceDataReader.php

```diff
@@ -125,4 +130,12 @@ public function setStepExecution(StepExecution $stepExecution)
     {
         $this->stepExecution = $stepExecution;
     }
+
+    protected function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setRequired([
+            'reference_data',
+            'ids'
+        ]);
+    }
```

Comment:
I think you've to check if the value in reference_data exists as reference data class or not